### PR TITLE
Ion Rangeslider

### PR DESCRIPTION
**Pull request by Maggie-M**

### CODE REVIEW

File: bokehjs/src/coffee/models/widgets/slidertemplate.eco [hunk 1]
```diff
@@ -1,5 +1 @@
-<label for="<%= @id %>"> <%= @title %>: </label>
-<input type="text" id="<%= @id %>" readonly style="border:0; color:#f6931f; font-weight:bold;">
-<div class="bk-slider-<%=@orientation%>">
-  <div class="slider " id="<%=@id%>">
-</div>
+<input type="text" class="js-range-slider" value="" />
```
Comment (mattpap) [inline bokehjs/src/coffee/models/widgets/slidertemplate.eco]: If this remains untemplated one-liner, remove this and change `SliderView.tagName` to `input` and use `@$el` directly with `ion-rangeslider`.

**Comment by mattpap** [comment]
@Maggie-M, despite this work being WIP, "Initial commit" isn't exactly helpful, especially this is the title of this PR.

**Comment by bryevdv** [comment]
@mattpap my fault I'm afraid I suggested it just to get a shareable PR up asap

**Comment by bryevdv** [comment]
Closing this for now, we can return to it later.

**Pull request closed without merging**